tripper/backends/sparqlwrapper.py

```diff
@@ -38,14 +40,17 @@ class SparqlwrapperStrategy:
     def __init__(
         self,
         base_iri: str,
+        update_iri: "Optional[str]" = None,
         username: "Optional[str]" = None,
         password: "Optional[str]" = None,
         **kwargs,
     ) -> None:
         kwargs.pop(
             "database", None
         )  # database is not used in the SPARQLWrapper backend
-        self.sparql = SPARQLWrapper(endpoint=base_iri, **kwargs)
+        self.sparql = SPARQLWrapper(
+            endpoint=base_iri, updateEndpoint=update_iri, **kwargs
+        )
```

Comment:
```suggestion
        )
                import warnings
        if not self.sparql.isSparqlUpdateRequest() and self.sparql.updateEndpoint is None:
            warnings.warn("The 'base_iri' is not a valid update endpoint. "
                    "For updates it is necessary to give the 'update_iri'.")
```

How about something like this? Might be better to put it in the update methods, so that not send out warnings when we do not want to do updates.... And the import should be moved to the right place, of course.